src/StyleSheet.js

```diff
@@ -30,22 +29,26 @@ export default class StyleSheet {
     this.rules = Object.create(null)
     // Used to ensure correct rules order.
     this.rulesIndex = []
-    this.classes = Object.create(null)
+    this.classes = this.options.classes = Object.create(null)
```

Comment:
Another option is to implement a setClasses method in RulesContainer and set the classes from the parent, but this is a bit of an overhead for this purpose.